source/Quantities/Units/ISystemInject.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Quantities.Dimensions;
+
+namespace Quantities.Units;
+
+public interface ISystemInject<out TDimension>
```

Comment:
I assume this interface will handle type injections, just like in the Repo you demoed.